inventory/src/main/java/nl/teamtwee/bep3/restaurant/inventory/infrastructure/driver/web/InventoryController.java

```diff
@@ -23,18 +23,19 @@
 import javax.validation.Valid;
 import javax.validation.constraints.Pattern;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
 
 @RestController
+@AllArgsConstructor
 @RequestMapping("/inventory")
 public class InventoryController {
-
     private final IngredientsCommandHandler commandHandler;
     private final IngredientsQueryHandler queryHandler;
 
-    public InventoryController(IngredientsCommandHandler commandHandler, IngredientsQueryHandler queryHandler) {
-        this.commandHandler = commandHandler;
-        this.queryHandler = queryHandler;
+    @PostMapping("/stock/take")
```

Comment:
niet nodig om de link `/stock` en `/take` is.

Reply:
?

Reply:
`@PostMapping("/stock")`

Is genoeg, `/take` hoeft niet